src/boundary_module.c

```diff
@@ -328,33 +328,33 @@ int apply_absorber_v2(  size_t N_x, size_t N_y, size_t N_z, int D_absorb,
     double
         damp;
 
-    // z1 absorber: z=0...d_absorb
+    // z1 absorber: z=0...D_ABSORB
 //#pragma omp parallel for collapse(2) default(shared) private(k,j,damp) <-- can collapse be used here?
     if ( strstr(absorber,"z1") ) {      
 #pragma omp parallel for default(shared) private(ii,jj,kk,damp)
         for (ii=2 ; ii<N_x-2 ; ii+=2) {
             for (jj=2 ; jj<N_y-2 ; jj+=2) {
-                for (kk=2 ; kk<D_absorb-2 ; kk+=2) {
-                    damp = ((double)kk-(double)D_absorb)/(double)D_absorb;
+                for (kk=2 ; kk<d_absorb-2 ; kk+=2) {
+                    damp = ((double)kk-(double)d_absorb)/(double)d_absorb;
                     damp = ABSORBER_DAMPING(eco,damp);
 
                     EB_WAVE[ii+1][jj  ][kk  ] *= damp;
                     EB_WAVE[ii  ][jj+1][kk  ] *= damp;
                     EB_WAVE[ii  ][jj  ][kk+1] *= damp;
 //                if ((ii%10 == 0) && (jj%10 == 0) && (kk%10 == 0)) 
 //                    printf( "z1: ii=%3d, jj=%3d, kk=%3d, (kk-d_abs)/d_abs=%f, damp=%f\n", 
-//                            ii, jj, kk, ((double)kk-(double)d_absorb)/(double)d_absorb, damp );
+//                            ii, jj, kk, ((double)kk-(double)D_ABSORB)/(double)D_ABSORB, damp );
```

Comment:
if I'm not mistaken this should read "d_absorb" here, as we are inside of the function "apply_absorber_v2" (the usefullness of defining it for that function separately is a different question though)